src/main/java/net/snowflake/client/core/HttpClientSettingsKeyBuilder.java

```diff
@@ -0,0 +1,63 @@
+package net.snowflake.client.core;
+
+public class HttpClientSettingsKeyBuilder {
```

Comment:
It might be more idiomatic to make this a static inner class of HttpClientSettingsKey just called `public static class Builder` instead of separate file.

Reply:
Done